wandb/apis/public.py

```diff
@@ -2605,8 +2628,17 @@ def get(
         }
         variables.update(kwargs)
 
-        response = client.execute(query, variable_values=variables)
-        if response.get("project") is None:
+        response = None
+        try:
+            response = client.execute(query, variable_values=variables)
+        except Exception:
+            # Don't handle exception, rely on legacy query
+            query = cls.LEGACY_QUERY
+            response = client.execute(query, variable_values=variables)
+
+        if response is None:
+            return None
```

Comment:
Im surprised we need this check now, do you know why this is required?

Reply:
`None.get()` excepts right?

Reply:
```
>>> response = None
>>> response.get("project")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'get'
```